### ISSUE TITLE
Death Unrespawnable

### ISSUE THREAD

**Issue by BLSK** [ForgeEssentials/ForgeEssentialsMain#1574]
It's taken me a week to figure out what was going on as another mod keeps taking credit for crashes and well...  When I die I get the respawn message  Problem is respawning gives me a Internal Server Error.  I had More Health Enhanced enabled at first which caused an endless loop of deaths...not sure if its crash report could be relevant but its
https://github.com/eidehua/More-Health-Enhanced/issues/2

Newest FE build from 6/9/15 used without More Health Enhanced
http://pastebin.com/76ASssKi

I am also trying to set the Starting Spawn point to Dimension -500 which I have a Mystcraft World that is dedicated to be spawn.  It isn't seeming to Lock in Right as newplayers still get sent to Dimension 0.  Maybe a goof up on my part but not 100% sure
Anyways...
Beta 7 from minecraft curse seems to work right.

**Comment by olee** [comment]
So in the tests before you didn't use latest build?
If you did not yet, use the latest build from http://ci.forgeessentials.com/job/FE/

**Comment by BLSK** [comment]
(have to run to work soonish)
I was actually, I updated to beta 8 like a week ago or so and was having no issues with deaths since I tend not to die and right now its just me doing most of the testing.
A friend hopped on and we went to 'the end' and he was surprised there were tons of weeping angels and lycanite mobs all over and a 60k hp dragon.  I told him the end was scary and he wanted to try it out... 1 second later the server crashed and it took a few days to find out what was going on and then I took up and died in the end...'death unrespawnable'  I took the report to the mod designer it specified it was for thinking I was done and if the person felt like updating it, great.  I only recalled about an hour before I wrote the message above, since I had chosen to look at another mod that was newer, 'Internal Server Error' specifying FE.  I backed off to beta 7 after but thought it would be great to give you guys the heads up, that respawning in my case was causing issues.  May of only happen with my unique pack, but yeah.

Anyways...

When I get home in aroundish 8 hours I will be trying the newer builds I see were made since my message.  I don't even recall at the moment why I updated 8 in the first place as it has been a long week, but yes... I was using the b8.736 at the time of 'Death Unlimited' and 'Death Kick', an interaction between More Health Enchanted and FE was causing Death Unlimited... but without Death Kick, Now having no issues back on beta 7...at least yet XD... I have broken a lot of things in the last 7 months.  The other day weeping angels was updated to fix an issue I had, just to crash all that updated after the fix.

anyways...

BLSK
